### DISCUSSION TITLE
[Kavita] Adding a zoom feature to the web browser

### DISCUSSION THREAD

**Discussion by Stevonthy** [Kareadita/Kavita#3138] [Ideas]
### Idea Description

Adding a little zoom bubble so that you don't need to use browser extensions and such would be amazing for small text 
and because kavita has the original file size the zoom bubble quality will be higher than if I just used a zoom extension!
(It could also be a square for all I care)
would also be nice if I could change the size of the zoom bubble (or square)

It could be 
![image](https://github.com/user-attachments/assets/b7ae1ae5-63c0-4749-8292-5156e596b5bf)
![image](https://github.com/user-attachments/assets/ee62a364-d6fc-46a4-b07b-d0daaf600c7b)

### Idea Category

User Experience

### Duration of Using Kavita

2 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by singularity098** [comment]
Second this!

I have recently switched to Kavita from using Foliate locally. The only feature that I miss is having an easy way to zoom in closer on some of the smaller panels and their text.

**Comment by majora2007** [comment]
FYI Zoom does work on mobile when not using Fullscreen mode.

**Comment by singularity098** [comment]
Thanks!  That's good to know.

My preferred way to consume manga has come to be a desktop web browser, though... on either my PC monitor or living room television with the HTPC, and I still struggle to read a few panels here and there.

**Comment by lpcvoid** [comment]
It would be perfect if zooming worked in fullscreen mode. That way I could read a comic book on my phone, and I'd be able to zoom in to text boxes.
